tests/dummy/app/templates/pie-chart.hbs

```diff
@@ -0,0 +1,10 @@
+{{!-- {{#ember-sparkles as |chart|}}
```

Comment:
I commented out the tests section because i was a bit overzealous when I started and generated files and started writing code. I commented out the code so that the build doesn't break while I write the add-on, right now I'm going to finish writing the add-on and then start working on the test.